src/patternfly/components/Page/examples/Page.md

```diff
@@ -15,7 +15,7 @@ wrapperTag: div
       {{#> page-header-brand-toggle}}
```

Comment:
looks like there is one more we need to add the href attr to for a variation that was just added under main section variations - https://github.com/patternfly/patternfly/blob/582d77fd842592483768ceed275175b89bb533dc/src/patternfly/components/Page/examples/Page.md#main-section-variations